examples/runHscTest.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+set -e
+
+# Setup
+# We don't want threading
+export OMP_NUM_THREADS=1
+# But we do want to process images in parallel on multiple cores.
+# Need ~2GB/core
+# Actually, 1 GB should be fine, but if you run out of memory on a VM
+# or compute node your job might just get killed without notice,
+# so 2GB/core to be safe.
+# The default here of 4 cores means this works on your laptop or an 8 GB VM.
+# If you're running on a larger node, you might increase this up to 16.
+N_CORES=4
```

Comment:
Ah, yes.  Very good!
Also changed to NUMPROC for consistency with the more general `runExample.sh` used for the other processing drivers.